crypto/func/builtins.cpp

```diff
@@ -1063,6 +1063,8 @@ void define_builtins() {
                       AsmOp::Nop());
   define_builtin_func("~dump", TypeExpr::new_forall({X}, TypeExpr::new_map(X, TypeExpr::new_tensor({X, Unit}))),
                       AsmOp::Custom("s0 DUMP", 1, 1), true);
+  define_builtin_func("~strdump", TypeExpr::new_forall({X}, TypeExpr::new_map(X, TypeExpr::new_tensor({X, Unit}))),
+                      AsmOp::Custom("STRDUMP", 1, 1), true);
```

Comment:
Please add "STRDUMP" to Asm.fif

Reply:
https://github.com/ton-blockchain/ton/blob/35d17249e6b54d67a5781ebf26e4ee98e56c1e50/crypto/fift/lib/Asm.fif#L1170 since those debug OPs are defined in white paper they already defined in `Asm.fif`, but just don't have realization. 

Thanks for review, I'll fix all shortly.